services/cookbook/cookbook.service.js

```diff
@@ -7,6 +7,9 @@ const { version: versionColor } = require('../../lib/color-formatters')
 
 // For Chef cookbook.
 module.exports = class Cookbook extends LegacyService {
+  static get url() {
```

Comment:
Does it help if you use the old-style:
````
const ServiceTester = require('../service-tester')

const t = new ServiceTester({ id: 'cookbook', title: 'Cookbook' })
module.exports = t
````

rather than `const t = createServiceTester()`?

Reply:
The old-style should help. But we used approach with `createServiceTester` + `static get url` in previous PRs: https://github.com/badges/shields/pull/2175, https://github.com/badges/shields/pull/2191/
Maybe we should pick one and document it in https://github.com/badges/shields/blob/master/doc/service-tests.md?

Reply:
To be honest as it only impacts legacy services, I don't think we should document it nor be too worried. If we've done this in other places, I'm happy to go with this approach here as well. 😉